src/utils/index.js

```diff
@@ -4,13 +4,16 @@ const gamesAPI = axios.create({
     baseURL: "https://ikes-games.cyclic.app/api"
 });
 
-export const getReviews = (category, sort_by) => {
+export const getReviews = (category, sort_by, order) => {
     let path = '/reviews?'
```

Comment:
This should be refactored to use axios params on the request config. It's in the NC Notes, but the docs give a good demonstration as well. Basically, if any of these values are undefined, they won't be used to create the path